bin/node/runtime/src/lib.rs

```diff
@@ -284,6 +289,11 @@ impl pallet_staking::Trait for Runtime {
 	type SlashCancelOrigin = pallet_collective::EnsureProportionAtLeast<_3, _4, AccountId, CouncilCollective>;
 	type SessionInterface = Self;
 	type RewardCurve = RewardCurve;
+	type NextSessionChange = Babe;
+	type ElectionLookahead = ElectionLookahead;
+	type Call = Call;
+	type SubmitTransaction = TransactionSubmitterOf<Self::KeyType>;
+	type KeyType = pallet_babe::AuthorityId;
```

Comment:
so as discussed, we use the babe keys (same as used for block authoring) for signing and verifying everything here.

Reply:
tbh I think im-online and staking should have no keys of their own and just use whatever the validators are using for their block authoring. 

What is the reason to have separate key ? (aside form being similar to other modules)

Code-wise, it is a minor change so no big deal anyway.

Reply:
We have introduced application keys exactly for making every "application" (Babe, Parachain, etc) use its own specific key.

Reply:
You also say it was discussed, where was this discussed?

Reply:
Was somewhere in a group chat. Can't dig it up now. I am fine with both if having a new key type is more conventional

Reply:
Yeah, `im-online` is using its own keys.

Reply:
I declare surrender and will add app keys :D